pytorch_object_detection/detection_models/__init__.py

```diff
@@ -0,0 +1,66 @@
+import torchvision
+import torch
+from torchvision.io.image import read_image
+from torchvision.models import WeightsEnum
+
+from torchvision.utils import draw_bounding_boxes
+from torchvision.transforms.functional import to_pil_image
+
+
+class DetectionPipeline:
```

Comment:
Is it base class? Should it be abstract?

Reply:
done

Reply:
Still not abstract.
Add load_model(...) abstract method to set up self.model and self.weights in inherited classes